goerli/373463.json

```diff
@@ -0,0 +1 @@
+[{"message":{"validator_index":"373463","from_bls_pubkey":"0x8d924ec8b4cf4752f5ad56337dd1205ad4b05f647b0ffa98a5921602ff8bd60d238b15796f4e42c8021931a39ebb6f9e","to_execution_address":"0x5385eefdae761edc311f08474a6ca0bf2eb8f2a7"},"signature":"0xb8f95dcf2e3efd7f024570997ca2ce60aeb28414d1bb64a60923128ececc6c43581e097c06fed45484a9d37c532412ae15e7d12f978c232990dd7880db2f7f3afb0b4f9192894710bfd551457f80a8a4cb7af46634be266129e6daaa56286e2a"}]
```

Comment:
The signature found in this file is incorrect for goerli/373463. A valid signature for validator 373463 is listed in your other file (goerli/373450.json), but it uses a different execution address. Please try again?